compute_tools/src/compute.rs

```diff
@@ -728,7 +728,10 @@ impl ComputeNode {
 
         // Write new config
         let pgdata_path = Path::new(&self.pgdata);
-        config::write_postgres_conf(&pgdata_path.join("postgresql.conf"), &spec, None)?;
+        let postgresql_conf_path = pgdata_path.join("postgresql.conf");
+        config::write_postgres_conf(&postgresql_conf_path, &spec, None)?;
```

Comment:
Not related to this PR, but just noticed that this will basically erase `extension_server_port` in the Postgres config at reconfiguration, which can break something if it can be changed on SIGHUP

Reply:
It can only be changed on postmaster, so shouldn't be an problem.
Agree that the code around `extension_server_port` can be cleaner...